src/Cli/dotnet/commands/dotnet-workload/install/WorkloadManifestUpdater.cs

```diff
@@ -194,6 +214,17 @@ public static void AdvertiseWorkloadUpdates()
             }
         }
 
+        public string GetAdvertisedWorkloadSetVersion()
+        {
+            var advertisedPath = GetAdvertisingManifestPath(_sdkFeatureBand, new ManifestId(WorkloadSetManifestId));
```

Comment:
There's only ever one advertising manifest for workload sets? I don't remember for sure.